README.md

```diff
@@ -23,104 +23,332 @@
+This repository is maintained by Googlers but is not a supported Firebase product. Issues here are answered by
+maintainers and other community members on GitHub on a best-effort basis.
 
 # Enable framework-awareness
 
 This is an experimental addon to the Firebase CLI to add web framework support, to enable it call the following:
 
 ```shell
-firebase --open-sesame frameworkawareness
+firebase experiments:enable webframeworks
 ```
 
-# Deploy Next.js
+# Prerequisites
+
+- Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
+
+### Serve locally
+
+You can test your integration locally by following these steps:
+
+1. Run `firebase serve` from the terminal. This should build your Angular app and serve it using the Firebase CLI.
+2. Open your web app at the local URL returned by the CLI (usually http://localhost:5000)
+
+### Deploy your app to Firebase Hosting
+
+When you're ready to share your changes with the world, deploy your Angular app to your live site:
+
+1. Run `firebase deploy` from the terminal.
+2. Check your website on: `SITE_ID.web.app` or `PROJECT_ID.web.app` (or your custom domain, if you did setup one)
 
-Easily deploy your Next.js application to Firebase and serve dynamic content to your users.
+## Guide: "Integrate Angular"
+
+Easily deploy your Angular application to Firebase and serve dynamic content to your users.
 
 ## What you'll need before you begin
 
 ### Prerequisites
+
 - Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
 - (optional) Billing enabled on your Firebase Project (if you plan to use SSR)
+- Optional: AngularFire
 
 ### Initialize Firebase
-To get started, you'll need to initialize Firebase for your framework project. Use the Firebase CLI for a new project, or modify `firebase.json` for an existing project.
+
+To get started, you'll need to initialize Firebase for your framework project. Use the Firebase CLI for a new project,
+or modify `firebase.json` for an existing project.
 
 #### Initialize a new project
 
-Run the initialization command from the CLI:
-    ```shell
-    firebase init hosting
-    ```
+1. Run the initialization command from the CLI: ``
+
+```shell
+firebase init hosting
+```
+
+1. Choose your hosting source directory; this could be an existing Angular App
+1. Choose "Dynamic web hosting with web framework"
+1. Choose Angular.
 
 #### Initialize an existing project
 
-Change your hosting config in firebase.json to have a `source` option, rather than a `public` option. For example:
+Change your hosting config in `firebase.json` to have a `source` option, rather than a `public` option. For example:
 
 ```json
 {
   "hosting": {
-    "source": "."
+    "source": "./path-to-your-angular-workspace"
   }
 }
 ```
 
-### Serve locally
+## Serve static content
```

Comment:
I think this should be h3, if "Integrate Angular" stays h2 (as opposed to becoming a page title or h1).

Reply:
Updated!